bin.src/genDefectRegistry.py

```diff
@@ -62,7 +66,7 @@ def __init__(self, path, version, validStart, validEnd=None):
 # Fix up end dates so there are no collisions.
 # Defects files for a CCD are valid from the date they are registered until the next date.
 # This means that any defects file should carry ALL the defects that are present at that time.
-for ccd, rowList in rowsPerCcd.iteritems():
+for ccd, rowList in rowsPerCcd.items():
     rowList.sort(key=lambda row: row.validStart) # ISO-8601 will sort just fine without conversion from str
```

Comment:
For reference, I think this would be a more modern version:

```python
from operator import attrgetter
rowList.sort(key=attrgetter('validStart'))
```

But since this executable code doesn't have any tests, I'm loathe to change it.